pages/about.html

```diff
@@ -1,12 +1,38 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
-</head>
-<body>
-  
-</body>
-</html>
\ No newline at end of file
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Amel Ntamark | About</title>
+    <meta name="description" content="about amel ntamark and how to contact" />
+    <link href="../styles/style.css" rel="stylesheet" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css?family=Roboto+Mono"
+    />
+  </head>
+  <body>
+    <nav id="nav-bar">
+      <ul>
+        <li><a href="index.html">home</a></li>
+        <li><a href="portfolio.html">portfolio</a></li>
+        <li><a href="about.html">about</a></li>
+      </ul>
+    </nav>
+    <header id="about-header"><h1>about</h1></header>
+    <div id="about-content">
+      <p>
+        currently, i'm a student at ada developer's academy studying fullstack web development. i have experience with python, javascript, C, SQL and postgres, HTML, CSS, and RESTful APIs. before discovering my passion for programming, i worked in nonprofit.<br> <br>
+        in my free time i like to write prose, songwrite, read, watch films, and hang out with loved ones. i'm a cameroonian girl born and raised between the pacific northwest and the UK.<br><br> 
+        favorite animal: monkeys and apes<br><br>
+        favorite color: purple<br><br>
+        favorite show: the good place<br><br>
+        favorite song: <iframe style="border-radius:12px" src="https://open.spotify.com/embed/track/29d0nY7TzCoi22XBqDQkiP?utm_source=generator" width="100%" height="80" frameBorder="0" allowfullscreen="" allow="autoplay; clipboard-write; encrypted-media; fullscreen; picture-in-picture"></iframe>
```

Comment:
Style: Using a `ul` or simple paragraphs instead of having a bunch of `br`s would be cleaner.  

From MDN:
> Creating separate paragraphs of text using `<br>` is not only bad practice, it is problematic for people who navigate with the aid of screen reading technology. Screen readers may announce the presence of the element, but not any content contained within `<br>`s. This can be a confusing and frustrating experience for the person using the screen reader.

To read more: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br

Reply:
Thank you for sharing this info Kaida! ✨